packages/components-css/logo/index.scss

```
@@ -0,0 +1,79 @@
+// LOGO STYLING
+.frameless-logo__wrapper {
+  align-items: center;
+  container: logowrapper / inline-size;
+  display: flex;
+  flex-basis: min-content;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.frameless-logo__text {
+  display: inline-block;
+  font-family: var(--frameless-logo-text-font-family);
+  font-size: var(--frameless-logo-text-font-size);
+}
+
+.utrecht-logo {
+  transform: scale(0.75);
+}
+
+// RESPONSIVENESS
+
+// KLEIN (60px - 185Ppx) | Alleen SVG 
+@container logowrapper (min-width: 60px) and (max-width: 185px) {
+  .frameless-logo__text {
+    display: none;
+  }
+
+  .utrecht-logo {
+    transform: scale(0.55);
+  }
+}
+
+// MEDIUM 1 (151px - 250px) | Text + SVG klein (flex:column)
+
+// @Media voor flex column
+@media (186px <= width <= 337px) {
+  .frameless-logo__wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 0.2rem;
+  }
+}
+
+// @Container voor Text + Logo klein
+@container logowrapper (min-width: 186px) and (max-width: 300px) {
+  .utrecht-logo {
+    transform: scale(0.55);
+  }
+
+  .frameless-logo__text {
+    font-size: 2rem;
+    margin-block: 0;
+    margin-block-end: 1rem;
+    margin-inline: 0;
+  }
+}
+
+// MEDIUM 2 (251px - 299px) | Text + SVG klein (flex:row) 
+@container logowrapper (min-width: 301px) and (max-width: 374px) {
+  .utrecht-logo {
+    transform: scale(0.55);
+  }
+
+  .frameless-logo__text {
+    font-size: 2em;
+  }
+}
+
+// LARGE (300px - 399px) | Text + SVG groot 
+@container logowrapper (min-width: 375px) {
+  .utrecht-logo {
+    transform: scale(0.75);
+  }
+
+  .frameless-logo__text {
+    font-size: 3em;
```

Wat is de reden om hier `em` te gebruiken en geen `rem`?

Ik zou trouwens verwachten dat je hier de custom property aan zou passen niet de css-selector.
```suggestion
    --frameless-logo-text-font-size: 3em;
```